target/linux/ipq40xx/patches-5.4/901-arm-boot-add-dts-files.patch

```diff
@@ -45,6 +45,10 @@ Signed-off-by: John Crispin <john@phrozen.org>
  	qcom-ipq4019-ap.dk04.1-c3.dtb \
  	qcom-ipq4019-ap.dk07.1-c1.dtb \
  	qcom-ipq4019-ap.dk07.1-c2.dtb \
++	qcom-ipq4019-deco-m5-eu-v3.0.dtb \
++	qcom-ipq4019-a62.dtb \
++	qcom-ipq4019-cm520-79f.dtb \
++	qcom-ipq4019-ea8300.dtb \
```

Comment:
This looks unrelated.

Reply:
AFAIK they are unrelated. Since it's a patch file, github is showing green lines where there shouldn't be... the only line that was actually added is line 48. 49, 50 and 51 just moved down one line.

Reply:
I don't think so, these lines are duplicated.